truss/cli/cli.py

```diff
@@ -471,6 +470,7 @@ def deploy(
                     break
                 elif num_failed := num_services - num_active - num_deploying:
                     break
+                time.sleep(2)
```

Comment:
Great call. nit: making `2` constant (ie: WAIT_BETWEEN_STATUS_CHECK), could make the intent clearer here